### PR TITLE
[zh-cn]: update the translation of Text `wholeText` property

### PR DESCRIPTION
**Pull request by T34-active**
### Description
https://developer.mozilla.org/en-US/docs/Web/API/Text/wholeText

### Related issues and pull requests

* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/text/wholetext/index.md
* Last commit: https://github.com/mdn/content/commit/0c8a320b035cf625c1df67713a94ead2e7f3aec6

### CODE REVIEW

File: files/zh-cn/web/api/text/wholetext/index.md [hunk 1]
```diff
@@ -1,86 +1,59 @@
...
+你决定不喜欢中间的句子，所以把它去掉了：
 
 ```js
-assert(para.firstChild.wholeText == "Thru-hiking is great!    However, ");
+const paragraph = document.querySelector("p"); // 读取该段
+paragraph.removeChild(paragraph.childNodes[1]); // 删除 strong 元素
 ```
 
-`wholeText` 只是文本节点的一个属性，特可以返回连接了所有相邻 (i.e. 没有被其他元素边界分开) 文本节点数据的字符串。
+现在你的结果是 _“徒步旅行很不错！然而，投票是一件棘手的事情。_，超链接前有两个节点：
```
Comment (T34-active) [inline files/zh-cn/web/api/text/wholetext/index.md:38]:
```suggestion
现在你的结果是 _“徒步旅行很不错！然而，投票是一件棘手的事情。"_，超链接前有两个节点：
```

File: files/zh-cn/web/api/text/wholetext/index.md [hunk 2]
```diff
@@ -1,86 +1,59 @@
...
-`wholeText` 只是文本节点的一个属性，特可以返回连接了所有相邻 (i.e. 没有被其他元素边界分开) 文本节点数据的字符串。
+现在你的结果是 _“徒步旅行很不错！然而，投票是一件棘手的事情。"_，超链接前有两个节点：
 
-现在让我们回到最初的问题。我们想做的是用新的文本替代旧的文本。这就是 {{domxref("Text.replaceWholeText", "replaceWholeText()")}} 用处所在：
-
-```js
-para.firstChild.replaceWholeText("Thru-hiking is great, but ");
-```
+1. 包含 {{domxref("Text")}} 字符串的 `"徒步旅行很不错！"`
+2. 第二个 `Text` 节点包含字符串 `"然而，"`
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/text/wholetext/index.md:41]:
```suggestion
1. 包含字符串 `"徒步旅行很不错！"` 的 {{domxref("Text")}} 节点
2. 第二个 `Text` 节点，包含字符串 `"然而，"`
```

File: files/zh-cn/web/api/text/wholetext/index.md [hunk 3]
```diff
@@ -1,86 +1,59 @@
...
 
-我们移除了所有的相邻文本节点 (所有构成 whole text 的文本节点) 除了调用`replaceWholeText()` 的，并且把剩余的文本改成了新文本。我们现在所得到的是这样的：
+要同时获取这两个节点，你需要调用 `paragraph.childNodes[0].wholeText`：
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/text/wholetext/index.md:43]:
```suggestion
要同时获取这两个节点，你可以调用 `paragraph.childNodes[0].wholeText`：
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Some suggestions

**Review by JasonLamv-t — APPROVED** [review]
👍

**Pull request merged**